### PR TITLE
PHOENIX-5852 The zkConnectionString in LoadBalance is incorrect

### PR DESCRIPTION
**Pull request by ZhaoBQ**
[PHOENIX-5852](https://issues.apache.org/jira/browse/PHOENIX-5852)

### CODE REVIEW

**Comment by infraio** [comment]
Add a ut for this?

**Comment by ZhaoBQ** [comment]
Hi @infraio , a ut has been added.

File: load-balancer/src/main/java/org/apache/phoenix/loadbalancer/service/LoadBalanceZookeeperConfImpl.java [hunk 1]
```diff
@@ -65,8 +65,23 @@ public String getServiceName(){
 
         @Override
         public String getZkConnectString(){
-            return String.format("%s:%s",configuration.get(QueryServerProperties.ZOOKEEPER_QUORUM_ATTRIB,
-                    "localhost"),configuration.get(QueryServerProperties.ZOOKEEPER_PORT_ATTRIB,"2181"));
+            String[] serverHosts = configuration.getStrings(QueryServerProperties.ZOOKEEPER_QUORUM_ATTRIB, "localhost");
+            String clientPort = configuration.get(QueryServerProperties.ZOOKEEPER_PORT_ATTRIB, "2181");
+            StringBuilder zkConnectionString = new StringBuilder();
```
Comment (stoty) [inline load-balancer/src/main/java/org/apache/phoenix/loadbalancer/service/LoadBalanceZookeeperConfImpl.java:70]: Why not just call ZKConfig.standardizeZKQuorumServerString() from HBase?
We are pulling in hbase-common anyways.

File: load-balancer/src/it/java/org/apache/phoenix/end2end/LoadBalancerEnd2EndIT.java [hunk 2]
```diff
@@ -139,6 +142,37 @@ public void testSingletonPropertyForLoadBalancer(){
         Assert.assertTrue(" the load balancer is not singleton",loadBalancer == anotherloadBalancerRef );
     }
 
+    @Test
+    public void testZkConnectString() {
+        // server port is different from the port which is set by user
```
Comment (stoty) [inline load-balancer/src/it/java/org/apache/phoenix/end2end/LoadBalancerEnd2EndIT.java:147]: Nit Technically, this is a UT, but we tend not worry about the distinction much.

**Comment by ZhaoBQ** [comment]
Fixed as you said. Please review @stoty

**Comment by stoty** [comment]
Committed. Thanks for the fix @ZhaoBQ

**Pull request closed without merging**